game.cpp

```diff
@@ -717,6 +750,39 @@ void test_game() //!OCLINT tests may be many
       }
   }
 
+  // Player enters cool down status after one shooting action
```

Comment:
```suggestion
  // (513) Player enters cool down status after one shooting action
```

We try to refer to issues names in the tests :)
This makes it easier to cross-reference issues and test if someone doesn't understand why it's there later on

Reply:
I will follow the convention of the code here